openprescribing/common/measure_tags.json

```diff
@@ -51,6 +51,10 @@
     "name": "Infections",
     "description": "These measures are for treatments commonly used for treating infections, including antibiotics and dressings."
   },
+  "iif": {
+    "name": "NHS England Impact and Investment Fund",
+    "description": "The <a href=\"https://www.england.nhs.uk/wp-content/uploads/2020/02/update-to-the-gp-contract-agreement-2021-2324.pdf\">Investment and Impact Fund</a> is an NHS England scheme that rewards Primary Care Networks (PCN) for delivering objectives in the NHS Long Term Plan. These measures are all included in the IIF annd PCNs can use them to review the success of any interventions they make. Please note that some of the prescribing measures (e.g. gastroprotective prescribing) in the IIF are not displayed here as the NHS does not publish the information. "
```

Comment:
Typos and readability improvements:

`(PCN)` -> `(PCNs)`
`annd` -> `and`
`Please note that some of the prescribing measures (e.g. gastroprotective prescribing) in the IIF are ` -> `Please note that some of the prescribing measures in the IIF (e.g. gastroprotective prescribing) are `

Can we say that we display N of M IIF measures?  e.g. `Please note that 4 of the 25 prescribing measures `

Reply:
Re N of measures. I agree but still waiting for publication from NHS England on what exactly is in the list from low priority. Will update with N and update text more generally with a link to a blog once we get that confirmation.